src/DataFixtures/AppFixtures.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: grantas
+ * Date: 18.4.8
+ * Time: 17.38
+ */
+
+namespace App\DataFixtures;
+
+use App\Entity\Course;
+use Doctrine\Bundle\FixturesBundle\Fixture;
+use Doctrine\Common\Persistence\ObjectManager;
+use Doctrine\ORM\Mapping\ClassMetadata;
+
+class AppFixtures extends Fixture
+{
+    public function load(ObjectManager $manager)
+    {
+            $courses = [
+                [
```

Comment:
Ateičiai siųlau naudoti associated array's tokiems atvejams, paskui kai settini duomenis vietoj `setTitle($courses[1][0])`galėsi rašyt `setTitle($courses['title'][0])`

Reply:
Be to, mano siųlymas ateity būtų iš vis pakeist duomenų struktūrą į:
```
[
  [
    'id' => 3,
    'title' => foo,
    # ...
  ],
  [
    # ...
  ]
]
```